client-go/controller/client/auth.go

```diff
@@ -129,16 +129,12 @@ func Passwd(username string, password string, newPassword string) error {
 		return err
 	}
 
-	resBody, status, err := client.BasicRequest("POST", "/v1/auth/passwd/", body)
+	_, err = client.BasicRequest("POST", "/v1/auth/passwd/", body)
```

Comment:
This could be collapsed with the `if err != nil` statement below, same for lines 150 and 182.